coda/coda_replication/factories.py

```diff
@@ -0,0 +1,23 @@
+"""
+Coda Replication Model factories for test fixtures.
+"""
+from datetime import datetime
+
+import factory
+from factory import fuzzy
```

Comment:
Can we join these two and just do something like `from factory import fuzzy, Sequence` and then change line 13 to be `ark = Sequence(lambda n: 'ark:/00001/id{0}'.format(n))`?